README.md

```diff
@@ -4,7 +4,9 @@ The Visier Python Connector allows Python developers to query against the Visier
 
 ## Prerequisites
 This connector acts as a bridge between your Python application and Visier's cloud-hosted service infrastructure. To successfully connect to your Visier data, you need:
-* The URL domain name prefix. For example: `https://{vanity-name}.api.visier.io`.
+* The URL domain name prefix. For example: `https://{vanity-name}.api.visier.io`. To find your vanity name:
+  1. In Visier, in the global workspace, click **Settings > Single Sign-On** or **Partner Single Sign-On**.
```

Comment:
Nit: should we separate `Settings` from the next option as `Settings` applies to both options?